pkg/fpgacontroller/patcher/patcher_test.go

```diff
@@ -25,12 +24,9 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 
 	fpgav2 "github.com/intel/intel-device-plugins-for-kubernetes/pkg/apis/fpga/v2"
+	"k8s.io/klog/v2/ktesting"
 )
 
-func init() {
-	_ = flag.Set("v", "4")
```

Comment:
Is the verbosity set somewhere else or was this an error?

Reply:
the default is 5 so I believe this is not needed.